### PR TITLE
Some js changes

### PR DESCRIPTION
**Pull request by Adel**

### CODE REVIEW

File: src/Collision.js [hunk 1]
```diff
@@ -32,13 +32,11 @@ export default {
                         eA.isKilled = true;
                     }
 
-                    if(eB.isFinisher) {
-                        gameIsWin = true;
-                    }
+                    gameIsWon = eB.isFinisher;
```
Comment (manland) [inline src/Collision.js]: :+1:

File: src/Renderer.js [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
-export const FORME_CIRCLE = 1;
-export const FORME_TRIANGLE = 2;
+export const FORMS = {
+    RECT: 0,
+    CIRCLE: 1,
+    TRIANGLE: 2
+};
+
+const formsFunc = {
+    0: (c, r) => c.rect(r.x, r.y, r.width, r.height),
```
Comment (manland) [inline src/Renderer.js]: minifier will minify, please lets it doing its job ;) c ==> Ctx & r ==> entity

File: src/Renderer.js [hunk 3]
```diff
@@ -62,5 +66,5 @@ window.addEventListener('resize', () => {
     canvasStatic.height = window.innerHeight * 0.8;
     canvasMovable.width = window.innerWidth * 0.8;
     canvasMovable.height = window.innerHeight * 0.8;
-    Renderer.init(level, canvasMovable, canvasStatic);
+    init(level, canvasMovable, canvasStatic);
```
Comment (manland) [inline src/Renderer.js]: Well done and i understand #50 ! need to talk about that tomorow ;)

File: src/Renderer.js (hunk 2 above)
Comment (Adel) [inline src/Renderer.js]: Reverted to earlier code as it was kinda useless!
Every "() =>" will be translated to "function{...}" so I am not sure we would gain anything with it!

**Pull request merged**